sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedIngestionSettings.cs

```diff
@@ -6,12 +6,14 @@
 namespace Azure.AI.MetricsAdvisor.Models
 {
     /// <summary>
+    /// Configures how a <see cref="DataFeed"/> behaves during data ingestion from its data source.
     /// </summary>
     public class DataFeedIngestionSettings
     {
         /// <summary>
         /// Creates a new instance of the <see cref="DataFeedIngestionSettings"/> class.
         /// </summary>
+        /// <param name="ingestionStartTime">The starting point in time from which data will be ingested from the data source. Subsequent ingestions happen periodically according to the specified granularity.</param>
```

Comment:
Is this the granularity we are referring to? 
```suggestion
        /// <param name="ingestionStartTime">The starting point in time from which data will be ingested from the data source. Subsequent ingestions happen periodically according to the specified <see cref="DataFeedGranularity"/>.</param>
```

Reply:
Yep.